relatio/_embeddings.py

```diff
@@ -239,3 +262,44 @@ def _load_keyed_vectors(self, model):
             raise
 
         return api.load(model)
+
+    def _get_default_vector(self, phrase: str) -> np.ndarray:
+
+        tokens = phrase.split()
+        embeddable_tokens = []
+        for token in tokens:
+            if token in self._vocab:
+                embeddable_tokens.append(token)
+            else:
+                warnings.warn(
+                    f"No vector for token: {token}. It is not used to compute the embedding of: {phrase}.",
+                    RuntimeWarning,
+                )
+        res = np.mean([self._model[token] for token in embeddable_tokens], axis=0)
+        return res
```

Comment:
We could try to avoid the duplication via the inheritance ... still not really a must.
This looks very similar to the lines before 227-239.

Reply:
Agreed, it would make the code neater

Reply:
Would you like to take care of this?